eras/shelley/impl/src/Cardano/Ledger/Shelley/AdaPots.hs

```diff
@@ -140,7 +140,7 @@ instance Show Produced where
 
 -- | Compute the Coin part of what is consumed by a TxBody, itemized as a 'Consume'
 consumedTxBody ::
-  ShelleyEraTxBody era =>
+  EraTxBody era =>
```

Comment:
Just curious, I can't see the link between the main change  (adding `ProtVerAtMost era 8` to  `ShelleyEraTxBody` ) and this - was this maybe overconstrained as it was?

Reply:
Yes. It was just overconstraint.